dask/dataframe/io.py

```diff
@@ -462,14 +462,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get if get else dask.get
```

Comment:
It was my understanding that dask.get always gets the "correct" scheduler. Is the following approach more robust?

```
from .context import _globals
_actual_get = _globals['get']
```

P.S.
This line is actually already in the code, it got in the previous to_hdf optimization PR, i'll gladly replace it with a better approach.